specs/derivation.md

```diff
@@ -996,13 +1010,13 @@ A deposit transaction is derived with the following attributes:
   - deploy new contract implementation
   - upgrade of `L1Block` through `ProxyAdmin`
 - `sourceHash`: `0x7dc74874297a8937186fdbec57ad344647a522de456088557e5fdeda88f66ddd`,
-  computed with the "Upgrade-deposited" type, with `intent = "Eclipse: L1Block upgrade"`
+  computed with the "Upgrade-deposited" type, with `intent = "Ecotone: L1Block upgrade"`
 
 Verify `sourceHash`:
 
 ```bash
 # compute intent hash:
-cast keccak "Eclipse: L1Block upgrade"
+cast keccak "Ecotone: L1Block upgrade"
 # 0x831b745c7397f93704ae55eb0100bf3c56fe9e304d3f21c1a93ec25f736fea26
```

Comment:
the digest is out of date

Reply:
whoops, search/replace gone wrong.  fixed.